src/migrations/1652114005124_create_table_users.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
```

Comment:
Already created the migrations for the tables. See '[18 tables and migrations definition](https://github.com/kelue/apexHauz/pull/20)'.